libraries/standard.js

```diff
@@ -81,29 +86,17 @@ const remove_if_empty = function (object, field) {
   }
 }
 
-const validate_params = function (params) {
-  // handle no passed options
-  if (!params) {
-    let empty_params = {};
-    empty_params.stringify = false;
-    empty_params.skip_errors = false;
-    empty_params.validate = false;
-    return empty_params;
-  }
-
-  // validation with schema
-  if (params.schema && !params.validator) {
+const validate_params = function (schema, validator) {
+  if (!schema && !validator) {
+    throw "no schema or validator provided"
+  } else if (schema && !validator) {
     throw "schema provided with no validator";
-  } else if (!params.schema && params.validator) {
+  } else if (!schema && validator) {
     throw "validator provided with no schema";
-  } else if (params.schema && params.validator) {
-    params.schema = JSON.parse(params.schema);
-    params.validate = true;
-  } else {
-    params.validate = false;
+  } else if (schema && validator) {
```

Comment:
this might be good as a first case (since we should assume that a schema and validator are given), or else this can just be an else case since it is the last case of 4 possible cases

Reply:
good catch